Week 1/16/Q2_16_Freddy.c

```diff
@@ -0,0 +1,21 @@
+//Q2. Write a program to input a positive number and reverse the digits of the number.
+
+#include<stdio.h>
+#include<conio.h>
+int main()
+{
+  int n, temp, s, r;
+  s = 0;
+  printf("Enter a number:");
+  scanf("%d", &n);
+  temp=n;
+  while(n>0)
```

Comment:
First check if the number is positive or not then find the reverse if it is positive